gluster-exporter/metric_brick.go

```diff
@@ -81,24 +130,45 @@ var (
 		Labels:    brickLabels,
 	})
 
-	subvolLabels = []MetricLabel{
-		{
-			Name: "volume",
-			Help: "Volume Name",
-		},
-		{
-			Name: "subvolume",
-			Help: "Sub volume name",
-		},
-	}
-
 	glusterSubvolCapacityUsed = newPrometheusGaugeVec(Metric{
 		Namespace: "gluster",
 		Name:      "subvol_capacity_used_bytes",
 		Help:      "Effective used capacity of gluster subvolume in bytes",
 		LongHelp:  "",
 		Labels:    subvolLabels,
 	})
+
+	glusterBrickLVSize = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "brick_lv_size",
+		Help:      "Bricks LV size MBytes",
+		LongHelp:  "",
+		Labels:    lvmLbls,
+	})
+
+	glusterBrickLVPercent = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "brick_lv_percent",
+		Help:      "Bricks LV usage percent",
+		LongHelp:  "",
+		Labels:    lvmLbls,
+	})
+
+	glusterBrickLVMetadataSize = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "brick_lv_metadata_size",
```

Comment:
Please convert this to bytes for uniformity and change the name of metric to `brick_lv_metadata_size_bytes`

Reply:
Ack